packages/webcomponents/src/components/business-forms/business-form/business-address/test/business-address.spec.tsx

```diff
@@ -1,7 +1,15 @@
 import { h } from '@stencil/core';
 import { newSpecPage } from '@stencil/core/testing';
 import { BusinessAddressForm } from '../business-address-form';
-import { BusinessAddressFormFields } from '../business-address-form-schema';
+
+interface BusinessAddressFormFields {
```

Comment:
I think we can import `iAddress` from `api/Business` here. Tried locally and Typescript was happy with it.